vm/src/stdlib/os.rs

```diff
@@ -2520,12 +2520,139 @@ mod posix {
                 sched_priority_repr.as_str()
             ))
         }
+
+        #[cfg(any(
+            target_os = "linux",
+            target_os = "netbsd",
+            target_os = "freebsd",
+            target_os = "android"
+        ))]
+        fn try_to_libc(&self, vm: &VirtualMachine) -> PyResult<libc::sched_param> {
+            let priority = self.sched_priority.clone();
+            let priority_type = priority.class().name();
+            let value = priority.downcast::<int::PyInt>().map_err(|_| {
+                vm.new_type_error(format!(
+                    "an integer is required (got type {})",
+                    priority_type
+                ))
+            })?;
+            let sched_priority = int::try_to_primitive::<i32>(value.as_bigint(), vm)?;
```

Comment:
The definition of `libc::sched_param::sched_priority` is `libc::c_int`. `i32` is the representation of building environement.
```suggestion
            let sched_priority = int::try_to_primitive::<libc::c_int>(value.as_bigint(), vm)?;
```

I think type inference will fill the type well even without it.

```suggestion
            let sched_priority = int::try_to_primitive(value.as_bigint(), vm)?;
```